scripts/prepare-commit.sh

```diff
@@ -14,8 +14,21 @@
 #                                                                         #
 ###########################################################################
 
+TOPLEVEL=$(git rev-parse --show-toplevel)
 
-PATH=$(git rev-parse --show-toplevel)/scripts:$PATH
+PATH=$TOPLEVEL/scripts:$PATH:$PWD/scripts
+
+if ! tty -s && [[ "$0" =~ /pre-commit ]]; then
+    exec </dev/tty
+fi
+
+cd "$TOPLEVEL"
```

Comment:
no push/popd ?

Reply:
It's a copy from qgis, I guess it should be good